sos/plugins/ovirt.py

```diff
@@ -114,6 +117,9 @@ def setup(self):
             "/etc/rhevm/",
             "/etc/ovirt-engine-dwh",
             "/etc/ovirt-engine-reports",
+            "/etc/ovirt-engine-metrics",
+            "/etc/ovirt-engine-setup",
+            "/etc/ovirt-vmconsole",
```

Comment:
Do we need to be doing any postprocessing for sensitive data in any of these directories?

Reply:
Hi, 
No, I did not add any sensitive data.